tool/src/kotlin/mod.rs

```diff
@@ -50,6 +50,7 @@ pub(crate) fn attr_support() -> BackendAttrSupport {
 struct KotlinConfig {
     domain: String,
     lib_name: String,
+    use_finalizers_instead_of_cleaners: Option<bool>,
```

Comment:
not a huge fan of the length of the name, perhaps `use_finalizers` is better, but no strong opinion, saying that this disables cleaners seems to be useful.

Reply:
hmm, you're right, but i do also like the info that it disables cleaners. i'm changing it to `use_finalizers_not_cleaners` , that's a bit shorter